source/clear-linux/get-started/cgdisk-manual-install/cgdisk-manual-install.rst

```diff
@@ -265,21 +265,40 @@ partitions and select to format them.
 
       Figure 14: :guilabel:`Set mount point of sda3`
 
-   Upon completion, the :guilabel:`Set mount points` appear as shown
-   in figure 15:
+#. Optional: Select :guilabel:`Encrypt Root Partition`, if
+   desired, as shown in figure 15. 
 
+   .. note:: 
+      `Set mount points` now show as completed. 
+   
    .. figure:: figures/cgdisk-manual-install-15.png
       :scale: 50 %
-      :alt: Set mount point completed
+      :alt: Encrypt root partition
+
+      Figure 15: :guilabel:`Encrypt root partition`
+
+#. Type a confirmation passphrase as directed.    

```

Comment:
Question on this:  Are there any rules that a user is required to follow for creating a passphrase?

Reply:
oh, I apologize, I completely missed this, I'm taking a look now

Reply:
@tpleavitt there is not rules for creating the passphrase